src/govuk/macros/i18n.njk

```diff
@@ -0,0 +1,13 @@
+{#
```

Comment:
Out of interest, is this following any sort of conventions for 'doc blocks' for Nunjucks macros, or is it something we've 'made up'?

Reply:
Just a thought – are we happy with this macro being part of the 'public API' for GOV.UK Frontend? It'd mean that if we made changes to it we'd need to treat them as breaking.

If not, we should try to be explicit about it being private. I don't believe Nunjucks has a concept of public/private for macros but we could for example rename it or put it in a folder called `_private` or something.

Reply:
That's a good shout, I'll prefix it with `_` and tag it `@private` in the doc which should make it clear that it's a function that we reserve to ourselves. I think I'd be keen to open it at the time we open the I18n API, as that'll be helpful for devs making their own macro to output attributes consistent with our API.

Reply:
So... tough luck, Nunjucks is not happy with importing `_` prefixed methods and complains with `names starting with an underscore cannot be imported` 😢  Tagging with `@private` in the doclet may be a bit light, but at least that we can do. 

I'm a bit confused about what makes and doesn't make our public API though. Shouldn't anything that's not documented on either the design system site or frontend one be deemed private/"use as your own risk we may break it" anyway?